pom.xml

```
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>11</jdk.version>
+        <jdk.version>17</jdk.version>
```

```suggestion
        <jdk.version>11</jdk.version>
```
I would suggest to remove all changes that you have done in pom.xml

if you code is not compiling on the local machine, check project settings

File | Project Structure | Project Settings | (check all tabs and set version 17)
or
File | Project Structure | Platform Settings | SDKs